src/worker_scripts/util.js

```diff
@@ -368,7 +368,8 @@
     self.webgazer.util.KalmanFilter.prototype.update = function(z) {
 
       // Here, we define all the different matrix operations we will need
-      var add = numeric.add, sub = numeric.sub, inv = numeric.inv, identity = numeric.identity;
+      // FIXME: mathjs is not exported to worker global scope  
+      var add = mathjs.add, sub = mathjs.subtract, inv = mathjs.inv, identity = mathjs.identity;
```

Comment:
I don't think this works since neither the `numeric` nor the `mathjs` variable is exported to the worker global scope. (as commented in https://github.com/brownhci/WebGazer/issues/163#issuecomment-861637706)

Reply:
Yeah, we need to fix that separately.